aleth-bootnode/main.cpp

```diff
@@ -132,9 +124,21 @@ int main(int argc, char** argv)
 
     Host h(c_programName, netPrefs, &netData);
     h.start();
+    if (!h.haveNetwork())
```

Comment:
What does it do? I prefer returning positive integers.

Reply:
We return -1 if there's no network since there's no point in proceeding - the Host can't do anything without the network. 

I chose to return -1 since that seems to be the standard error code returned in the various tools (aleth, aleth-vm, etc). To address this, I defined a new error enum (`EthErrors` since `Errors `is already taken) and return those values instead. I've defined the error enum in `libethcore\Common.h`...I'm not completely convinced that this is best place for it, do you know of a better place? Perhaps `libdevcore\Common.h`?

Reply:
Ok, then return -1 for now. The `EthErrors` must be moved to another PR. Rename to `AlethErrors`?

Reply:
Okay done, opened #5491

Reply:
@halfalicious this PR still has changes that went there I think

Reply:
Yes, because I hadn't pushed my updates yet 😄 Just force-pushed.